search-api.md

```diff
@@ -23,12 +23,6 @@ After receiving a request, the remote server can respond in one of two ways:
   * If a compatible version (`vX.Z` where `Z>=Y`) is supported by the remote server, it should provide a response using this version.
   * If no appropriate version is supported by the remote server, it should respond with `Not Acceptable (406)`, containing a JSON body with a description of the error. All responses, including this one, should contain a `Content-Type` header with the latest API version supported by the server. This will enable the user to re-submit the request using this version of the API.
```

Comment:
OK, I see the confusion point - I assumed that the phrase "All responses, including this one, should contain a `Content-Type` header with the latest API version supported by the server" includes the case of replies in supported versions other than the latest. If this only applies to unsupported versions I would rephrase it as "All error responses...."